### DISCUSSION TITLE
Allow the usage with URL on address bar

### DISCUSSION THREAD

**Discussion by pmsousa** [mjisaak/skilling-champion-extension#31] [Ideas]
Hi,

I would like to use this also with the address at the URL address bar.

Expected behavior:

- Right click address and copy the URL with the creator id appended.

**Comment by mjisaak** [comment]
Hi @pmsousa,

Thank you for your feedback. I have converted the issue to a discussion.
I just checked the chrome extension API and wasn't able to find a way to implement that. I will keep the discussion open, maybe someone will find a way to implement that. 

BR,
Martin

**Reply by pmsousa** [reply]
Hi @mjisaak,

Great job with this extension, it helps a lot.

I'm not knowledged on extension development, so I can't help but let's see if someone pick this up.

Pedro
